app/spec.html

```diff
@@ -1,16 +0,0 @@
-<!doctype html>
-<html>
-<head>
-    <meta charset="utf-8">
-    <title>Jasmine Spec Runner</title>
-
-    <link rel="stylesheet" type="text/css" href="vendor/jasmine/jasmine.css">
-
-    <script src="vendor/jasmine/jasmine.js"></script>
-    <script src="vendor/jasmine/jasmine-html.js"></script>
-    <script src="vendor/jasmine/boot.js"></script>
-    <script src="spec.js"></script>
-</head>
-<body>
-</body>
-</html>
```

Comment:
also 
* /tasks/build/generate_spec_imports.js
* `bundleSpecs` in  /tasks/build/build.js
* corresponding modules in package.json